src/lib.rs

```diff
@@ -6,5 +6,6 @@ pub mod keys;
 pub mod multienc;
 pub mod nonces;
 pub mod participant;
+pub mod samples;
```

Comment:
- Why `pub`?
- Why not `cfg(test)`?
- Why not including it in a `tests` submodule?